tools/dotnet-deb-tool/ArgumentEscaper.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dotnet.Deb.Tool
+{
+    public static class ArgumentEscaper
```

Comment:
I wish we just had the ability to pass a string[] args to process.Start...

Anyways, It's already duplicated 3 times, between utils and the two build_projects, do you think it'd be preferable to just take a dependency on CLI.Utils nuget package instead?